### PR TITLE
Upgrade common-ts to v2.0.3

### PR DESCRIPTION
**Pull request by tilacog**
Note to reviewers: 
Merge only after https://github.com/graphprotocol/common-ts/pull/106 is merged and published to NPM.

### CODE REVIEW

File: packages/indexer-native/package.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
     "clean": "rm -rf ./node_modules ./binary ./build ./coverage ./native/target ./native/artifacts.json ./native/index.node"
   },
   "dependencies": {
-    "@graphprotocol/common-ts": "2.0.1",
```
Comment (pcarranzav) [inline packages/indexer-native/package.json:49]: I hope touching this package doesn't break anything...

**Review by pcarranzav — APPROVED** [review]
LGTM assuming that CI passes when common-ts is deployed, and I'd suggest double checking indexer-native doesn't break things

File: packages/indexer-native/package.json (hunk 1 above)
Comment (tilacog) [reply packages/indexer-native/package.json:49]: Before merging, I'll check if the images built out of this PR work as expected.
If not, we will leave `indexer-native` untouched.

Comment (tilacog) [reply packages/indexer-native/package.json:49]: I've deployed images built out of this PR to our testnet, and both `indexer-service` and `indexer-agent` seem to be working as before. `indexer-native` build succeeded :)

**Pull request merged**